src/main/java/com/yegor256/xsline/StAfter.java

```diff
@@ -49,4 +49,23 @@ public StAfter(final Shift shift, final Shift next) {
         );
     }
 
+    /**
+     * Ctor.
+     * @param shift Original shift
+     * @param after Collection of Shifts to be applied after original Shift
+     */
+    public StAfter(final Shift shift, final Shift... after) {
```

Comment:
@MikhailLipanin I think that the ctor above can easily be deleted, since it is now just a specific case of this more generic ctor